winterize/init.lua

```diff
@@ -42,7 +42,7 @@ minetest.register_lbm({
 	end
 })
 
-local function can_snow_fall_freely(pos)
+local function snow_can_fall_freely(pos)
```

Comment:
Since the return values are swapped, why does this have to be renamed? Now it seems like the function assertively says that snow can indeed fall freely... :P

Reply:
`if snow_can_fall_freely then`?

Reply:
I don't want to be too picky, the current name is fine by me. :)

Reply:
Okie. It's all going over my head lol